### ISSUE TITLE
Remove usage of capabilities

### ISSUE THREAD

**Issue by crodriguezvega** [cosmos/ibc#1133]
Basically to keep up to date with the the [changes in ibc-go](https://github.com/cosmos/ibc-go/issues/7107)

**Comment by womensrights** [comment]
Closing this issue as seems to be completed by https://github.com/cosmos/ibc-go/pull/7279

**Comment by DimitrisJim** [comment]
This pertains to capability references in spec no? (Assuming this is why you reopened, though)

**Comment by womensrights** [comment]
yes I was going through the Eureka board and it was there so assumed it was for ibc-go, then realised it was a specs issue, but we don't need to do this for eureka launch so re-opened and removed from the project board

**Comment by tgrecojs** [comment]
Is there any documentation, discourse post, issue, etc. - any resource that dives into the topic of removing capabilities in greater depth? I would greatly appreciate getting eyes on that so I can see any discussions that led to the removal of capabilities 🙂
